sormas-backend/src/test/java/de/symeda/sormas/backend/user/UserServiceTest.java

```diff
@@ -72,42 +73,42 @@ public void testGetReferenceList() {
+		result = getUserService().getReferenceList(regionUuids, districtUuids, communityUuids, includeSupervisors, filterByJurisdiction, activeOnly, userRoles);
 		assertThat(result, contains(admin));
 
 		// 1b. Exclude admin by role
 		userRoles = Arrays.asList(UserRole.CASE_OFFICER);
-		result = getUserService().getReferenceList(regionUuids, districtUuids, includeSupervisors, filterByJurisdiction, activeOnly, userRoles);
+		result = getUserService().getReferenceList(regionUuids, districtUuids, communityUuids, includeSupervisors, filterByJurisdiction, activeOnly, userRoles);
 		assertThat(result, is(empty()));
 
 		// 2. Include supervisors
 		userRoles = Arrays.asList(UserRole.CONTACT_OFFICER);
 		RDCF rdcf = creator.createRDCF();
 		UserDto supervisor = creator.createUser(rdcf, UserRole.CONTACT_SUPERVISOR);
-		result = getUserService().getReferenceList(regionUuids, districtUuids, includeSupervisors, filterByJurisdiction, activeOnly, userRoles);
+		result = getUserService().getReferenceList(regionUuids, districtUuids, communityUuids, includeSupervisors, filterByJurisdiction, activeOnly, userRoles);
 		assertThat(result, is(empty()));
 		includeSupervisors = true;
-		result = getUserService().getReferenceList(regionUuids, districtUuids, includeSupervisors, filterByJurisdiction, activeOnly, userRoles);
+		result = getUserService().getReferenceList(regionUuids, districtUuids, communityUuids, includeSupervisors, filterByJurisdiction, activeOnly, userRoles);
 		assertThat(result, hasSize(1));
 		assertThat(result.get(0).getUuid(), equalTo(supervisor.getUuid()));
 
 		// 3. Exclude inactive user as overall condition
 		getUserFacade().disableUsers(Arrays.asList(supervisor.getUuid()));
-		result = getUserService().getReferenceList(regionUuids, districtUuids, includeSupervisors, filterByJurisdiction, activeOnly, userRoles);
+		result = getUserService().getReferenceList(regionUuids, districtUuids, communityUuids, includeSupervisors, filterByJurisdiction, activeOnly, userRoles);
 		assertThat(result, is(empty()));
 
 		// 4. filterByJurisdiction to test that the invocation works and filters correctly concerning activeOnly
 		filterByJurisdiction = true;
-		result = getUserService().getReferenceList(regionUuids, districtUuids, includeSupervisors, filterByJurisdiction, activeOnly, userRoles);
+		result = getUserService().getReferenceList(regionUuids, districtUuids, communityUuids, includeSupervisors, filterByJurisdiction, activeOnly, userRoles);
 		assertThat(result, is(empty()));
 		activeOnly = false;
-		result = getUserService().getReferenceList(regionUuids, districtUuids, includeSupervisors, filterByJurisdiction, activeOnly, userRoles);
+		result = getUserService().getReferenceList(regionUuids, districtUuids, communityUuids, includeSupervisors, filterByJurisdiction, activeOnly, userRoles);
 		assertThat(result, hasSize(1));
 		assertThat(result.get(0).getUuid(), equalTo(supervisor.getUuid()));
```

Comment:
please add a new test case for testing your changes

Reply:
Test was present with a bug. I just corrected the logic and removed 2 additional users from the test, in order to make it workable.

Reply:
please add a new test case for testing your changes

Reply:
Test is present under UserFacadeEjbTest.testGetReferencesByRoleAndJurisdiction()
It was present initially with bug. Than I corrected the code and the test itself.